src/test/node-unit-tests.js

```diff
@@ -39,13 +37,23 @@ const webpack = require("webpack");
 const webpackConfig = require("../../webpack.config");
 delete webpackConfig.entry.bundle;
 
-// The source map worker is compiled with webpack (and mock-require
-// doesn't work in workers) so mock it with an alias, and tweak a few
-// things to make the stub fetcher work in node.
-webpackConfig.resolve.alias["devtools-network-request"] =
-  "devtools-network-request/stubNetworkRequest";
+function stubNetworkRequest(url) {
+  return new Promise((resolve, reject) => {
+    const fs = require("fs");
+    path = require("path");
```

Comment:
When passed as a string and executed, this will pollute the global scope with `path`. Use `const _path` with a small comment and restore the `const path` for the require in the beginning of the file?